io.asgardio.tomcat.oidc.sample/src/main/webapp/error.jsp

```diff
@@ -0,0 +1,68 @@
+<%@ page import="io.asgardio.java.oidc.sdk.exception.SSOAgentException" %>
```

Comment:
Move imports after the licenses.

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-oidc-agent/pull/7/commits/bf4f7fa7367a16bd78c9cbabc954df5fb7451e18